srcpkgs/redis/template

```diff
@@ -1,41 +1,45 @@
 # Template file for 'redis'
 pkgname=redis
-version=7.0.5
+version=7.0.8
 revision=1
-makedepends="jemalloc-devel libatomic-devel"
+build_style=gnu-makefile
+make_build_args="V=1"
+make_check_target="test"
 checkdepends="pkg-config procps-ng tcl-devel which"
 short_desc="Advanced key-value store"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="BSD-3-Clause"
 homepage="https://redis.io"
-changelog="https://raw.githubusercontent.com/antirez/redis/${version%.*}/00-RELEASENOTES"
-distfiles="http://download.redis.io/releases/${pkgname}-${version}.tar.gz"
-checksum=67054cc37b58c125df93bd78000261ec0ef4436a26b40f38262c780e56315cc3
+changelog="https://raw.githubusercontent.com/redis/redis/${version%.*}/00-RELEASENOTES"
+distfiles="http://download.redis.io/releases/redis-${version}.tar.gz"
+checksum=06a339e491306783dcf55b97f15a5dbcbdc01ccbde6dc23027c475cab735e914
 
 system_accounts="redis"
 redis_homedir="/var/lib/redis"
 conf_files="/etc/redis/redis.conf"
 
 make_dirs="/var/lib/redis 0700 redis redis"
 
+if [ "$XBPS_TARGET_LIBC" != "musl" ]; then
+	makedepends+=" jemalloc-devel"
+	_malloc="jemalloc"
+fi
+
+make_build_args+=" MALLOC=${_malloc:-libc}"
+
+if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
+	makedepends+=" libatomic-devel"
+fi
+
 do_configure() {
```

Comment:
Please rename this `post_patch`. These modifications never beloned in the configure stage.